src/viewer/NoNetworkViewer.jsx

```diff
@@ -7,9 +7,11 @@ import styles from './styles'
 const NoNetworkViewer = ({ t, onReload }) => (
   <div className={styles['pho-viewer-canceled']}>
     <h2>{t('Viewer.loading.error')}</h2>
-    <Button theme="regular" onClick={onReload}>
-      {t('Viewer.loading.retry')}
-    </Button>
+    <Button
+      theme="regular"
```

Comment:
`theme="regular"` is no longer useful, as regular is the default theme. Not that it will break but it's just redundant.